db/schema.rb

```diff
@@ -10,12 +10,12 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20180518164135) do
+ActiveRecord::Schema.define(version: 2018_06_07_144046) do
```

Comment:
Rubocop told someone to do this, but it is maybe a lie, but ... mmm. maybe not. no idea. interesting.